app/controllers/application_controller.rb

```diff
@@ -1,3 +1,22 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  layout :layout_by_resource
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  private
+
+  def layout_by_resource
+    if user_signed_in?
+      'application'
```

Comment:
@thdurante, this line is not covered because we don't have any pages using `application` layout